x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/index.mock.ts

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
I created an [issue](https://github.com/elastic/kibana/issues/152471) to create a package with Cases mocks to remove this file and import the Case UI Service mock from the package.